doc/meetings/2023_02_17.md

```diff
@@ -0,0 +1,32 @@
+# Meeting 2023-02-17
+
+https://github.com/ElektraInitiative/PermaplanT/issues/37#issuecomment-1433926689
+
+## Questions from Lukas and Markus:
+
+1. Should we pin the rust version in any way, or is it ok to always use the latest version?
+    - we use latest
+2. How should we handle the situation during build? (no Postgres, etc.)
+ 
+3. How do we inject the correct `BASE_URL` for the backend into the frontend?
+
+    - .env file
+
+## Meeting notes:
+
+Updates from Markus:
+
+- there should be easy ways to indicate areas of interest which do not fit any layer
+- gardens of various sizes for beta testing
+- Yvonne started drawing for permaplant
+- Next friday set milestones with final dates
+
+Discussions:
+
+- Deployment in react https://create-react-app.dev/docs/deployment/
+- Both deployment methods are equal from our point of view. /api proxy vs static port
+- treffle.io is a big database but not much information
+- Diesel needs a database even on the compile time
+- 3 independent PSQL containers for PR, main and releases
```

Comment:
```suggestion
- 3 independent containers for PRs, master and releases
```

Reply:
I've taken the suggestions over, however, the word "containers" is ambiguous in my opinion e.g. docker is the first thing that comes to my mind. I suggest we rephrase or specify it in the future to avoid misunderstandings among the new members of the project

Reply:
Good idea! Specifically, we use the lightweight containers from Proxmox VE, which use [Linux Containers (LXC)](https://linuxcontainers.org/lxc/introduction/) see also https://pve.proxmox.com/wiki/Linux_Container